htdocs/css/bootstrap.css

```diff
@@ -3867,8 +3867,8 @@ textarea.input-group-sm > .input-group-btn > .btn {
 }
 
 .nav-tabs > li.active > a,
-.nav-tabs > li.active > a:hover,
-.nav-tabs > li.active > a:focus {
+.nav-tabs > li.active > a,
+.nav-tabs > li.active > a {
```

Comment:
we avoid changing bootstrap.css, and any other external dependencies, directly - which is the original reason we have bootstrap-rcloud-tweaks.css. I'm moving this rule to bootstrap-rcloud-tweaks.